doc/cci_guide.md

```diff
@@ -0,0 +1,1703 @@
+# 1. Test Objective
+The cci test is aimed to test CUBRID CCI driver. We write cci test cases to test all kinds of CCI Applications, such as `cci_connect()`, `cci_connect_with_url()`, `cci_cursor()`, `cci_fetch()`, `cci_get_data()`, `cci_get_result_info()` and so on. Actually cci test is a branch of shell test, the only difference is that cci shell scripts must execute C codes which include CUBRID CCI API.
+
+
+# 2. Execute CCI Test
+To perform the cci test, we need to install CTP first.   
+## 2.1 Install CTP
```

Comment:
Remove detail description for 2.1. 

Replace with:
Please refer to the guide to [install CTP in Linux platform]( https://github.com/CUBRID/cubrid-testtools/blob/develop/doc/ctp_install_guide.md#1-install-ctp-in-linux-platform).